core/src/main/java/org/infinispan/persistence/file/SingleFileStore.java

```diff
@@ -473,10 +473,26 @@ public boolean delete(Object key) {
          resizeLock.readLock().unlock();
       }
 
+      org.infinispan.commons.io.ByteBuffer keyBb;
+      org.infinispan.commons.io.ByteBuffer valueBb = null;
```

Comment:
I believe we should not declare this here and leave it below and just pass null on line 484 for both values.  Is a bit more clear imo.